forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt

```diff
@@ -0,0 +1,10 @@
+Name:Tishana's Tidebinder
+ManaCost:2 U
+Types:Creature Merfolk Wizard
+PT:3/2
+K:Flash
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounterAbility | TriggerDescription$ When CARDNAME enters the battlefield, counter up to one target activated or triggered ability. If an ability of an artifact, creature, or planeswalker is countered this way, that permanent loses all abilities for as long as Tishana's Tidebinder remains on the battlefield. (Mana abilities can't be targeted.)
+SVar:TrigCounterAbility:DB$ Counter | TargetType$ Activated,Triggered | TgtPrompt$ Choose target ability | RememberCountered$ True | ValidTgts$ Card,Emblem | SubAbility$ DBEffect | SpellDescription$ Counter target activated or triggered ability. If a permanent's ability is countered this way, activated abilities of that permanent can't be activated this turn. (Mana abilities can't be targeted.)
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STLoseAB | RememberObjects$ Remembered | ConditionDefined$ Remembered | ConditionPresent$ Card.Artifact,Card.Creature,Card.Planeswalker | ConditionCompare$ EQ1 | Duration$ UntilHostLeavesPlay
```

Comment:
bad Condition

Reply:
the Types were correct?

Reply:
Is it just the types+`inZoneBattlefield `?

Reply:
yea sorry, should have been clearer